assets/data/categories_en.json

```diff
@@ -2028,5 +2028,112 @@
       "Ygritte",
       "Yunkai"
     ]
+  },
+  {
+    "id": "MCU",
+    "image": "mcu.png",
+    "name": "Marvel Cinematic Universe",
+    "questions": [
+      "Avengers",
+      "Tony Stark",
+      "Iron Man",
+      "Steve Rogers", 
+      "Captain America",
+      "Bruce Banner",
+      "Hulk",
+      "Natasha Romanoff",
+      "Black Widow",
+      "Thor",
+      "Loki",
+      "Odin",
+      "Frigga",
+      "Hela",
+      "Mjölnir",
+      "Storm Breaker",
+      "Heimdall",
+      "Bifröst",
+      "Ragnarok",
+      "Valkyrie",
+      "Asgard",
+      "Sakaar",
+      "Nine Realms",
+      "Clint Barton",
+      "Hawkeye",
+      "Phil Coulson",
+      "Maria Hill",
+      "Nick Fury",
+      "Pepper Potts",
+      "Happy (Hogan)",
+      "James Rhodes",
+      "War Machine",
+      "Falcon",
+      "Sam Wilson",
+      "Wanda Maximoff",
+      "Scarlet Witch",
+      "Quicksilver",
+      "J.A.R.V.I.S.",
+      "Vision",
+      "Ultron",
+      "Stephen Strange",
+      "Peter Parker",
+      "Spider-Man",
+      "Bucky (James Barnes)",
+      "Winter Soldier",
+      "Civil war",
+      "Sokovia Accords",
+      "Guardians of the Galaxy",
+      "Star Lord",
+      "Peter Quill",
+      "Rocket",
+      "Groot",
+      "Gamora",
+      "Drax the Destroyer",
+      "Nebula",
+      "Mantis",
+      "The Collector",
+      "Knowhere"
+      "Thanos",
+      "The snap"
+      "T'Challa",
+      "Black Panther",
+      "Okoye",
+      "Shuri",
+      "Wakanda",
+      "Dora Milaje",
+      "Vibranium",
+      "Antman",
+      "Wasp",
+      "Scott Lang",
+      "Hope van Dyne",
+      "Hank Pym",
+      "Quantum realm",
+      "Chitauri",
+      "Kree",
+      "Skrull",
+      "Malekith",
+      "(Infinity) Gauntlet",
+      "Infinity stones",
+      "Power stone",
+      "Mind stone",
+      "Eye of Agamotto",
+      "Time stone",
+      "Aether",
+      "Tessaract",
+      "Space stone",
+      "Reality stone",
```

Comment:
You can add `Aether` here in brackets instead of placing it as separate word.

Reply:
Good point! They are combined now.